imports/plugins/included/product-detail-simple/client/components/variantList.js

```diff
@@ -20,7 +20,7 @@ class VariantList extends Component {
     }
   }
 
-  handleChildleVariantClick = (event, variant) => {
+  handleChildVariantClick = (event, variant) => {
```

Comment:
Good find, but should be in a separate PR as it doesn't really pertain to the Stripe Connect Integration